test/plugins/validation/2and3/operations-shared.js

```diff
@@ -715,5 +715,56 @@ describe('validation plugin - semantic - operations-shared', function() {
       );
       expect(res.warnings.length).toEqual(0);
     });
+
+    it('should complain about an unused tag', function() {
+      const spec = {
+        tags: [
+          {
+            name: 'some tag'
+          },
+          {
+            name: 'some other tag'
+          }
+        ],
+        paths: {
+          '/': {
+            get: {
+              operationId: 'get_everything',
+              tags: ['not a tag'],
+              summary: 'get everything as a string or an array',
+              responses: {
+                '200': {
+                  content: {
+                    'text/plain': {
+                      schema: {
+                        type: 'string'
+                      }
+                    },
+                    'application/json': {
```

Comment:
Will you simplify this responses object? No reason to have that in the test for tags and an array response should return an error, it only isn't because type is an empty string for some reason.

`summary` should be changed to match